### DISCUSSION TITLE
use image as a point on map

### DISCUSSION THREAD

**Discussion by ch4r1s** [Mapsui/Mapsui#2149] [Q&A]
I have been trying to add an image as a point on a map but no luck yet. For example I want to add an antenna on the map that I know its location. Is it possible for that point on the map instead of a circle point, to have an image be the point on that location?

**Comment by charlenni** [comment]
Take a look into the Symbols-Styles [sample](https://github.com/Mapsui/Mapsui/blob/master/Samples/Mapsui.Samples.Common/Maps/Styles/SymbolsSample.cs). There you could find such behavior.
